packages/api/src/graphql/article.ts

```diff
@@ -139,7 +139,7 @@ export const GraphQLArticleRevision = new GraphQLObjectType<ArticleRevision, Con
     socialMediaTitle: {type: GraphQLString},
     socialMediaDescription: {type: GraphQLString},
     socialMediaAuthors: {
-      type: GraphQLList(GraphQLAuthor),
+      type: GraphQLNonNull(GraphQLList(GraphQLAuthor)),
```

Comment:
Missing one `GraphQLNonNull` between list and author. same as with properties or tags.

Reply:
Done